apps/dashboard/scripts/migrate-clickhouse-pageviews.sh

```
@@ -0,0 +1,157 @@
+
+set -e
+
+echo "========================================"
+echo "ClickHouse Pageviews Migration"
+echo "========================================"
+echo ""
+
+# Load environment variables from root .env
+if [ -f "../../.env" ]; then
+  echo "Loading environment from ../../.env"
+  export $(cat ../../.env | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)